### PR TITLE
Adds an admin-only gun, because I'm fun like that 🔫

### PR DESCRIPTION
**Pull request by FlattestGuitar**
I'll use it in a bit, it's gonna be a Syndicate thing, maybe. We'll see.

Fires the glorious .50 cal rounds, deals 80 ~~damage~~ love per shot.

Also, has real weird sprites for the speedloaders (I think they look cool)

All the lovely sprites done by @scrubmcnoob

### CODE REVIEW

**Comment by Fox-McCloud** [comment]
Do we *really* need yet *another* admin-only gun; we've added, what? 3? 4? in the past couple months alone....and we already have a ton.

**Comment by Spartan6** [comment]
The Syndicate already have a revolver. The .357

While I love the model, perhaps try and incorporate it into the actual round...

File: code/modules/projectiles/ammunition/ammo_casings.dm [hunk 1]
```diff
@@ -99,6 +99,11 @@
 	caliber = ".45"
 	projectile_type = /obj/item/projectile/bullet/midbullet3
 
+/obj/item/ammo_casing/f500
+	desc = "A .50 bullet casing."
```
Comment (scrubmcnoob) [inline code/modules/projectiles/ammunition/ammo_casings.dm]: This is triggering my americanism. it should be A .500 magnum bullet casing.

**Comment by scrubmcnoob** [comment]
> Do we really need yet another admin-only gun; we've added, what? 3? 4? in the past couple months alone....and we already have a ton.

yes, i poured my artistic sweat and sole into this sprite

File: code/modules/projectiles/projectile/bullets.dm [hunk 2]
```diff
@@ -8,6 +8,10 @@
 	sharp = 1
 	hitsound_wall = "ricochet"
 
+/obj/item/projectile/bullet/strongbullet //ouch
+	damage = 80
+	stamina = 60
```
Comment (Fox-McCloud) [inline code/modules/projectiles/projectile/bullets.dm]: This is pretty much a one-shot, one-kill gun, at this stage.

It does 80 damage and instantly weakens them.

**Comment by Spacemanspark** [comment]
I have no problem with this.

**Comment by FlattestGuitar** [comment]
A great point. I did not consider the damage damage would tally along with stamina damage. It's a tad less omnilethal now, but it's still a straight upgrade from the .357 boolets.

We could potentially use it for events as well.... 🤔

**Comment by Spacemanspark** [comment]
Can we add an M4A1 next

**Comment by Allfd** [comment]
@Fox-McCloud 

We really like ballistic guns..... :gun::gun::gun::gun::gun::gun::gun:

Although we should probably add attachment points for flashlights on some of them 🤔

**Comment by scrubmcnoob** [comment]
flashlight+knife+pistol=mw2 tac knife usp. lets do it

**Comment by KingPhilipIII** [comment]
If it's admin only for Syndicake officers, it might as well be a meme and blow people away like nobody's business.

**Comment by tigercat2000** [comment]
@KingPhilipIII Nah, admin weapons are more fun when they do a reasonable amount of damage. It encourages admins to spawn it every once in a while for players that don't really need to be overkill, and besides, what fun is it without a little challenge to being robust?

**Comment by Anoonki** [comment]
>Although we should probably add attachment points for flashlights on some of them 

Flashlight rail on WT-550s and Riot Shotguns when?
550s just seem like a little tacticool SWAT weapon that'd be neat to have a gunlight on.
Riot shotguns (when not sawed off) take up your suit storage or a hand, so giving them the ability to mount a gunlight'd be pretty handy.

**Comment by jughu** [comment]
Why add ballastic guns that are admin only just add more customizable vars n shit so you could change a icon m such to make your own meme gun by adding a useless admin gun to the code

**Comment by Jovaniph** [comment]
What the point of admin only guns?

**Comment by Allfd** [comment]
@Jovaniph 

It lets us have fancy guns without having to deal with balance considerations.

**Comment by FlattestGuitar** [comment]
The idea here is that I'll be adding a few more admin-only items for events.

Dum is spriting a bunch of lovely SolGov jumpsuits and we want to make events have a bit more lore to them, which will be a ton of fun (I hope, at least) for all involved.

This gun is just a personal thingie I wanted to add, because poor Syndicate officers got nothing cool, while CC officers had their badass pulse pistols.

Just because this gun is admin-only, doesn't mean you won't ever see it on-station. It will just be very, very rare.

**Comment by Lady-Luck** [comment]
>Dum is spriting a bunch of lovely SolGov jumpsuits and we want to make events have a bit more lore to them, which will be a ton of fun (I hope, at least) for all involved.

How about a SolGov PR instead of a standalone admin-gun PR?

**Comment by FlattestGuitar** [comment]
ATOMIC PRs!!!!1!11!

Also, it's easier to review these one at a time and I don't trust myself to code it properly.

**Comment by Birdtalon** [comment]
I like it but I'm not an admin and therefore will likely never see it in game so I can't really support it.

**Comment by mrturkeytoe** [comment]
Perhaps add as a more expensive alternative to the .357 for warops

**Comment by tristan1333** [comment]
Let .357's be retextured like the detective's. Cool and slick. At least, that's what I'd like.

**Comment by FlattestGuitar** [comment]
I'd rather do that in another PR.

Though it would be neat if the mateba was in use instead of the ugly .357 we have right now.

**Comment by Fox-McCloud** [comment]
While interesting, it was not seen as necessary to add yet another weapon to the Administrators' already massive arsenal.

**Pull request closed without merging**